dev-support/hbase-personality.sh

```diff
@@ -589,13 +589,17 @@ function hadoopcheck_rebuild
     else
       hbase_hadoop2_versions="2.8.5 2.9.2 2.10.0"
     fi
-  else
-    yetus_info "Setting Hadoop 2 versions to test based on branch-2.3+/master/feature branch rules."
+  elif [[ "${PATCH_BRANCH}" = branch-2.* ]]; then
+    yetus_info "Setting Hadoop 2 versions to test based on branch-2.3+ rules."
     if [[ "${QUICK_HADOOPCHECK}" == "true" ]]; then
       hbase_hadoop2_versions="2.10.0"
     else
       hbase_hadoop2_versions="2.10.0"
     fi
+  else
+    yetus_info "Setting Hadoop 2 versions to test based on master/feature branch rules."
+    # we do not support hadoop 2 for hbase 3.x any more
```

Comment:
put this in the info message to make clear we're skipping Hadoop 2 because hbase 3+ won't use it.